domain/src/main/kotlin/no/nav/su/se/bakover/domain/brev/BrevInnhold.kt

```diff
@@ -17,7 +17,10 @@ abstract class BrevInnhold {
         val personalia: Personalia,
         val satsbeløp: Int,
         val fradragSum: Int,
-        val avslagsgrunn: Avslagsgrunn,
+        val fradrag: List<FradragPerMåned>,
+        val avslagsgrunner: List<Avslagsgrunn>,
+        val avslagsgrunnerLength: String = avslagsgrunner.size.toString(),
```

Comment:
Lurer på om jeg kanskje ville gjort denne til en Boolean (evt. en String-representasjon av en boolean, dersom det er problematisk for templaten med booleans). Og da kalt den noe sånt som `harFlereAvslagsgrunner` eller `kunEnAvslagsgrunn` litt avhengig av hvilken vei det blir enklest å bruke den i templaten.
Nå antar jeg forsåvidt bare at det er denne som er tenkt brukt til å bestemme om det er én eller flere avslagsgrunner i templaten. Eller er det viktig å vite antallet?

Reply:
heh ja. vet ikke helt hvorfor jeg ikke tenkte på det med det første😅